src/web/styles.css

```
@@ -0,0 +1,591 @@
+  font-family:
+    Inter,
+    ui-sans-serif,
+    system-ui,
+    -apple-system,
+    BlinkMacSystemFont,
+    "Segoe UI",
+    sans-serif;
+  color: #172033;
+  background: #edf2f3;
+  font-synthesis: none;
+  text-rendering: optimizeLegibility;
```

Stylelint (17.14.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)